spec/controllers/vacancies_controller_spec.rb

```diff
@@ -114,5 +114,26 @@
         expect(response).to have_http_status(:not_found)
       end
     end
+
+    context 'when using cookies' do
+      let(:vacancy) { create(:vacancy) }
+      let(:params) { { id: vacancy.slug } }
+      let(:vacancy_page_view) { instance_double(VacancyPageView) }
+
+      it 'should call the track method if cookies not set' do
+        expect(VacancyPageView).to receive(:new).with(vacancy).and_return(vacancy_page_view)
+        expect(vacancy_page_view).to receive(:track)
+
+        subject
+      end
+
+      it 'should not call the track method if smoke_test cookies set' do
+        expect(VacancyPageView).not_to receive(:new).with(vacancy)
+        expect(vacancy_page_view).not_to receive(:track)
```

Comment:
I think this expectation is redundant and having the one above it should be sufficient.

Reply:
Resolved in 36439d1